tests/dependencies.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: localhost
+  connection: local
+  remote_user: root
+  tasks:
+    - name: installing repo for Java 8 in Ubuntu
+      apt_repository: repo='ppa:openjdk-r/ppa'
+      when:
+        - ansible_os_family == 'Debian'
+        - ansible_distribution_version|version_compare(16.04, '<')
+
+- hosts: localhost
+  connection: local
+  remote_user: root
+  roles:
+    - role: geerlingguy.java
+      java_packages:
+        - openjdk-8-jdk
+      when: ansible_os_family == 'Debian'
+
+    - role: geerlingguy.java
+      java_packages:
+        - java-1.8.0-openjdk
+      when: ansible_os_family == 'RedHat'
```

Comment:
I'ld expect a solution involving less ansible boilerplate, to simply install a current (open)jdk-8 for different distros (here debian/ubuntu & redhat). or really not?